### PR TITLE
update for image info to quay

### PR DESCRIPTION
**Pull request by zhouying7780**
@kasturinarra  Please help review. 
run logs from my localhost: http://pastebin.test.redhat.com/925188

### CODE REVIEW

File: features/cli/status.feature [hunk 1]
```diff
@@ -31,8 +31,8 @@ Feature: Check status via oc status, wait etc
     Then the step should succeed
     When I run the :status client command
     Then the step should succeed
-    And the output should contain:
-      | pod/hello-openshift runs aosqe/hello-openshift |
+    And the output should match:
+      | pod/hello-openshift runs quay.io/openshifttest/hello-openshift.* |
```
Comment (liangxia) [inline features/cli/status.feature:35]: `.*` is not needed for faster match.

**Comment by kasturinarra** [comment]
@zhouying7780 could you please help check this PR and resolve conflicts if this is still valid ? Thanks !!

**Comment by zhouying7780** [comment]
@kasturinarra The feature has updated by other pr, so will close this one.

**Comment by kasturinarra** [comment]
> @kasturinarra The feature has updated by other pr, so will close this one.

Could you please help give me the PR if you have it handy, thanks !!

**Comment by zhouying7780** [comment]
@kasturinarra Didn't have the pr, just checked the code has used the quay image.

**Pull request closed without merging**